src/mediumComplexityProblem7/SortArray.java

```diff
@@ -0,0 +1,58 @@
+package mediumComplexityProblem7;
+
+public class SortArray {
+    public void ascendingSort(int[] inputArray) {
+        int firstCounter;
+        int secondCounter;
+        int temp = 0;
+
+        //Displaying elements of original array
+        System.out.println("Elements of original array: ");
+        for (firstCounter = 0; firstCounter < inputArray.length; firstCounter++) {
+            System.out.print(inputArray[firstCounter] + " ");
+        }
+
+        //Sort elements of original array in ascending order
+        for (firstCounter = 0; firstCounter < inputArray.length; firstCounter++) {
+            for (secondCounter = firstCounter + 1; secondCounter < inputArray.length; secondCounter++) {
```

Comment:
try to find the unique element and add or update the array in this for loop

Reply:
Done!!